charts/barman/templates/configmap-barman.yaml

```
@@ -19,15 +19,23 @@ data:
 
       barman_user = {{ .Values.barman.barmanUser }}
 
+      {{- if hasKey .Values.barman "postBackupRetryScript" }}
+      post_backup_retry_script = {{ .Values.barman.postBackupRetryScript }}
+      {{- end }}
+
+      {{- if hasKey .Values.barman "preRecoveryRetryScript" }}
+      pre_recovery_retry_script = {{ .Values.barman.preRecoveryRetryScript }}
+      {{- end }}
+
       ; barman_lock_directory = %(barman_home)s
       compression = {{ .Values.barman.compression }}
       configuration_files_directory = /etc/barman/barman.d
       last_backup_maximum_age = {{ .Values.barman.lastBackupMaximumAge }}
-      minimum_redundancy = 1
+      minimum_redundancy = 0
```

Is there a specific reason why you would want to change this to default to 0?
```suggestion
      minimum_redundancy = 1
```